common-test-lib/src/main/kotlin/com/google/container/tools/test/ContainerToolsRule.kt

```diff
@@ -133,6 +133,7 @@ class ContainerToolsRule(private val testInstance: Any) : TestRule {
                 }
             }
 
+            file.setExecutable(true)
```

Comment:
It does not seem necessary for each test file we are creating, could we add a boolean parameter (optional, defaults to false) to mark file as executable? Or some other way to mark file executable in the unit test itself?